src/main/java/com/google/sps/data/SmallCityService.java

```diff
@@ -19,14 +36,27 @@ public SmallCityService(MapLocation mapLocation) {
   }
 
   public void findAllBusinesses() {
-    // TODO: Get businesses from Place API given user location
     businesses = new LinkedList<Listing>();
-    businesses.add(new Listing("LA Fitness", new MapLocation(40.457091, -79.915331), 3.9, null, "https://www.lafitness.com/Pages/Default.aspx"));
-    businesses.add(new Listing("west elm", new MapLocation(40.456279, -79.915015), 3.6, null, "https://www.westelm.com"));
-    businesses.add(new Listing("McDonalds", new MapLocation(40.459450, -79.918479), 2.6, null, "https://www.mcdonalds.com"));
-    businesses.add(new Listing("East End Brewing Company", new MapLocation(40.459391, -79.911782), 4.7, "https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.eastendbrewing.com%2F&psig=AOvVaw0kX_SAlxhA09EN3cKpt5ik&ust=1593613487774000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCLDA6oDfqeoCFQAAAAAdAAAAABAD", "http://www.eastendbrewing.com/"));
-    businesses.add(new Listing("The Shiny Bean Coffee & Tea", new MapLocation(40.496328, -79.944862), 4.9, "https://goo.gl/maps/AYH2QCL7pkoMBxHA8", "https://theshinybean.com/"));
-    businesses.add(new Listing("Weisshouse", new MapLocation(40.456684, -79.925499), 4.3, "https://goo.gl/maps/7tuXn7QF2hh7ioGYA", "https://www.weisshouse.com/"));
+    LatLng latLng = new LatLng(user.getGeolocation().lat, user.getGeolocation().lng);
+    final GeoApiContext context = new GeoApiContext.Builder()
+            .apiKey(KEY)
+            .build();
+    NearbySearchRequest request = PlacesApi.nearbySearchQuery(context, latLng);
```

Comment:
This can be abstracted from the smallcity service into a separate object to handle business related logic. Please connect with @AbakirH on this.

Reply:
We talked about this and are a bit confused? Do you mean that I would abstract out essentially this entire function to a different class and call a new function through the BusinessesService class?

Reply:
Yes. Doing this for business related logic, and map / geolocation logic etc, will help keep this class simple and easy to parse. 

*I think refactor may have actually been the correct word here, rather than abstract. Apologies.

Reply:
Ah okay I see. Just created a BusinessesService class and moved the code over. Let me know if this is what you wanted!

Reply:
Yes. That works.